config/manager/manager.yaml

```diff
@@ -103,6 +103,7 @@ spec:
         - --logtostderr=true
         - -v=5
         - --feature-gates=AllAlpha=true,AllBeta=true
+        - --max-workers-for-pull-image=2
```

Comment:
- --max-workers-for-pull-image=5

Reply:
Setting it to 2 makes it more convenient for validation and end-to-end testing.